src/main/java/dev/slimevr/vr/processor/skeleton/HumanSkeleton.java

```diff
@@ -1684,34 +1635,24 @@ public void resetSkeletonConfig(SkeletonConfigOffsets config) {
 				{
 					skeletonConfig.setOffset(SkeletonConfigOffsets.TORSO, null);
 				}
-				break;
-			case CHEST: // Chest is 57% of the upper body by default (shoulders
+			}
+			case CHEST -> // Chest is 57% of the upper body by default
+							// (shoulders
 				// to chest)
```

Comment:
This comment is split weirdly, just some visual annoyance
```suggestion
			case CHEST -> // Chest is 57% of the upper body by default
							// (shoulders to chest)
```